### ISSUE TITLE
Add to opening_brace regex

### ISSUE THREAD

**Issue by gregoryloden** [realm/SwiftLint#1821]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Rule Request

Version 0.22.0

With `opening_brace`, code like
```
if condition1,
    condition2
{
```
is flagged. It seems like it should work just as well as
```
if
    condition1,
    condition2
{
```
